config/postgresql.xml

```diff
@@ -3,6 +3,7 @@
     <connection>
         <pg_host>localhost</pg_host>
         <pg_port>5432</pg_port>
+        <pg_azure>false</pg_azure>
```

Comment:
Flexible server is in preview right now. We want to support benchmarking databases that are in production. Since we have is_azure in SQL server I followed the naming convention. Would is_azure_single_server a better name? 

I'm not sure if removing CreateUserDatabase will affect the benchmark or not